[Big Sur] Fix Big Sur more after 251860@main

litherum opened the pull request:
#### d3586113e535030963e2f30aa8702d27753d5457
<pre>
[Big Sur] Fix Big Sur more after 251860@main
<a href="https://bugs.webkit.org/show_bug.cgi?id=242147">https://bugs.webkit.org/show_bug.cgi?id=242147</a>

Unreviewed.

Turns out 11 and 12 are different numbers.

* Source/WTF/wtf/PlatformHave.h:

Canonical link: <a href="https://commits.webkit.org/251971@main">https://commits.webkit.org/251971@main</a>
</pre>

aproskuryakov commented on Source/WTF/wtf/PlatformHave.h:
```diff
@@ -1018,11 +1018,7 @@
-    || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 140000) \
-    || (PLATFORM(APPLETV) && __TV_OS_VERSION_MAX_ALLOWED >= 140000) \
-    || (PLATFORM(WATCHOS) && __WATCH_OS_VERSION_MIN_REQUIRED >= 70000)
+#if PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000
```
This doesn't look right to me, do you want `#if !PLATFORM(MAC) || __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000`?

litherum replied:
You're right that this is probably better.

litherum committed:
[Big Sur] Fix Big Sur more after 251860@main
https://bugs.webkit.org/show_bug.cgi?id=242147

Unreviewed.

Turns out 11 and 12 are different numbers.

* Source/WTF/wtf/PlatformHave.h:

Canonical link: https://commits.webkit.org/251971@main
